Sources/MD5.swift

```diff
@@ -0,0 +1,274 @@
+/*
```

Comment:
What's the advantage of bring this in vs using a library?

Reply:
I don't understand this question completely. Which library did you have in mind?

Reply:
Can you add that last sentence as a code comment in the top of this file? That makes the motivation clearer without needing to search github/commit messages.